pages/plotly-express.py

```diff
@@ -3,18 +3,20 @@
 from utils.code_and_show import example_app
 
 
-dash.register_page(__name__, description="Plotly Express scatter matrix")
+dash.register_page(__name__, description="Plotly Express scatter matrix example with top-down layout. This app contains adding components to graph via dropdown. ")
 
 filename = __name__.split("pages.")[1]
 
 
 notes = """
+#### Dash Components in App:
+- [Markdown](https://dash.plotly.com/dash-core-components/markdown)
+- [Dropdown](https://plotly.com/python/dropdowns/)
+- [Scatter Matrix](https://plotly.com/python/splom/)
```

Comment:
- Please remove Markdown. I can't find it in the file.
- Update the dropdown link to the dropdown component link from Dash Docs because this is the dash docs section.
- Scatter Matrix link is from Plotly documentation. Please move this link under the Plotly Doc section